src/main/java/io/kontur/disasterninja/dto/ActiveSubscriptionRequestDto.java

```diff
@@ -0,0 +1,28 @@
+package io.kontur.disasterninja.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.UUID;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class ActiveSubscriptionRequestDto {
+    @NotNull(message = "appId is required")
+    private UUID appId;
+
+    @NotBlank(message = "billingPlanId is required")
+    @Size(min = 26, max = 26, message = "billingPlanId must be exactly 26 characters long")
```

Comment:
This is PayPal-specific, isn't it? We'll have to revisit it if another payment gateway has different limits...

Reply:
yes, for sure, they will be more general than.